core/services/job/orm.go

```diff
@@ -684,10 +684,7 @@ func (o *orm) FindJobs(offset, limit int) (jobs []Job, count int, err error) {
 			return err
 		}
 		for i := range jobs {
-			err = o.LoadEnvConfigVars(&jobs[i])
-			if err != nil {
-				return err
-			}
+			err = multierr.Combine(err, o.LoadEnvConfigVars(&jobs[i]))
```

Comment:
Out of scope, but `LoadEnvConfigVars` is a misnomer now, since it assumes env vars. Could generalize the name to just be about config.